lib/features/mailbox_dashboard/presentation/widgets/advanced_search/advanced_search_filter_form.dart

```diff
@@ -22,86 +22,82 @@ class AdvancedSearchInputForm extends GetWidget<AdvancedFilterController>
   @override
   Widget build(BuildContext context) {
     controller.initSearchFilterField(context);
-    return SingleChildScrollView(
-      child: Padding(
-        padding: const EdgeInsets.symmetric(horizontal: 28, vertical: 12),
-        child: Material(
-          color: Colors.white,
-          child: Column(
+    return Material(
```

Comment:
I don't think it is a good idea when you remove `ScrollView`? is it ok with any size of window?

Reply:
Let me check web again. Mobile still scrollview